src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -174,10 +174,34 @@ func (w logglyErrorWriter) Write(b []byte) (int, error) {
 		"timeZone":  w.tz,
 		"version":   w.versionToLoggly,
 	}
+	fullMessage := string(b)
+
+	// extract last 2 (at most) chunks of fullMessage to message, without prefix,
+	// so we can group logs with same reason in Loggly
+	parts := strings.Split(fullMessage, ":")
+	var message string
+	pl := len(parts)
+	switch pl {
+	case 1:
+		message = ""
+	case 2:
+		message = parts[1]
+	default:
+		message = parts[pl-2] + ":" + parts[pl-1]
+	}
+	message = strings.Trim(message, " \n")
```

Comment:
You could use strings.TrimSpace here to trim all leading and trailing whitespace.